src/components/HTMLEngineProvider/HTMLRenderers/ImageRenderer.tsx

```diff
@@ -113,14 +111,4 @@ function ImageRenderer({tnode}: ImageRendererProps) {
 }
 
 ImageRenderer.displayName = 'ImageRenderer';
-
-export default withOnyx<ImageRendererProps, ImageRendererWithOnyxProps>({
-    user: {
-        key: ONYXKEYS.USER,
-    },
-})(
-    memo(
-        ImageRenderer,
-        (prevProps, nextProps) => prevProps.tnode.attributes === nextProps.tnode.attributes && prevProps.user?.shouldUseStagingServer === nextProps.user?.shouldUseStagingServer,
-    ),
-);
+export default ImageRenderer;
```

Comment:
Please move the logic from ImageRendererWrapper file to this file
```suggestion
export default ImageRendererWrapper;
```